pkg/datafetcher/atmos_fetcher.go

```diff
@@ -0,0 +1,26 @@
+package datafetcher
+
+import (
+	_ "embed"
+	"errors"
+)
+
+//go:embed schema/atmos-manifest.json
+var atmosSchema string
```

Comment:
The issue here is there is an atmos stack schema, an atmos config schema, a vendor schema, a component schema, and a workflow schema. 

This assumes we have one schema for Atmos, which we do not.